cmd/osbuild-mock-openid-provider/main.go

```diff
@@ -0,0 +1,148 @@
+package main
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"flag"
+	"log"
+	"math/big"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/golang-jwt/jwt"
+)
+
+const jwtPublicKeyPEM = `
+-----BEGIN PUBLIC KEY-----
+MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAv4FiKP8EBOjIdqkQJAcn
++TjE6w7KPLghlnSMlrGk1PZ1K8CHWYj1xymGYQDhLe0QGtBCZv0+wI1UjAI3xCMU
+n+5siufiI6HrtE0ZCUzxEArQMKYxnmZMnl9nuirBevOyFyhx39o2n1iARXTOBrtB
+OuWrC1D1EqGyuhS/Tc/Wf/lpKZDCTUwWrfBRxQL1IDKyh5bBWCc9bGoIvPu3Xfdh
+FMbK8VRsEw5jq3smKQc7mE4rO3jjJWIyLe09vtVZ3lMP301Ud+cawelTnKhJPMkY
+IJl8V39V6f44NjRBvsLoQjPIAVvm4PdvXdzZnqwexr8+vT0Vf3U/klxmbT+2yDgG
+uwIDAQAB
+-----END PUBLIC KEY-----
+`
+
+const jwtPrivateKeyPEM = `
```

Comment:
Hmm, @jabia99 lately worked on removing all hard-coded test secrets from the repository. I wonder if we should also remove this one otherwise we're basically "regressing". Sorry, I know how painful is to generate keys. :(

Reply:
You can take a look at tools/gen-ssh and do something similar with this key. If you prefer, I could do it myself if you leave the hard coded key like it is.

Reply:
[Just did it inline](https://github.com/osbuild/osbuild-composer/pull/1717/files#diff-8c9fc1f11806f990402d945e7a6a3bdcf8be429120a4b8c2ec18e9ff5251f02fR1177-R1178).